File: mlkit-smartreply/app/src/main/java/com/google/firebase/samples/apps/mlkit/smartreply/MainActivity.java
package com.google.firebase.samples.apps.mlkit.smartreply;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.google.firebase.samples.apps.mlkit.smartreply.ui.main.MainFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_activity);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, MainFragment.newInstance())
                    .commitNow();
        }
    }
}

File: mlkit-smartreply/app/src/main/java/com/google/firebase/samples/apps/mlkit/smartreply/kotlin/MainActivity.kt
package com.google.firebase.samples.apps.mlkit.smartreply.kotlin

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import com.google.firebase.samples.apps.mlkit.smartreply.R
import com.google.firebase.samples.apps.mlkit.smartreply.kotlin.chat.ChatFragment

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main_activity)

        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                    .replace(R.id.container, ChatFragment.newInstance())
                    .commitNow()
        }
    }
}
